lib/site_prism/section.rb

```diff
@@ -81,5 +82,9 @@ def parent_page
       candidate = candidate.parent until candidate.is_a?(SitePrism::Page)
       candidate
     end
+
+    def wait_until_invisible(*args, **kwargs)
```

Comment:
I like the way this is done inside the section proper to lean on this, but I feel this is going to be extended and extended over time. I think maybe for now this is ok though